### ISSUE TITLE
PackerSync not running on save

### ISSUE THREAD

**Issue by GCBallesteros** [Blovio/NeoVim_Essentials#2]
https://github.com/Blovio/NeoVim_Essentials/blob/82b3847baa5d3ce9107e23eacccd34a76e6c1303/nvim/lua/plugins/init.lua#L17

The line above won't run on save because the file expected for the autocommand is `plugins.lua` but that is not the name of the relevant file here (`nvim/lua/plugins/init.lua`).

I modified that line on my config to:
```vim
autocmd BufWritePost ~/.config/nvim/lua/plugins/init.lua source <afile> | PackerSync
```

I'm sure there is a more general way using env variables but this did it for me.

P.S. Thanks for the config, I'm using it as the basis for my hybrid(lua+vimscript) -> lua migration!

**Comment by GCBallesteros** [comment]
Closing, this has been fixed later in the day by commit `e26a4834f`.

**Comment by Blovio** [comment]
Hey, thanks for the update, I'll leave this here in case people want to use your method. 

```
vim.cmd([[
  augroup packer_user_config
    autocmd!
    autocmd BufWritePost ~/.config/nvim/lua/plugins/init.lua source <afile> | PackerSync
  augroup end
]])
```

I ended up removing it because I think it goes against the minimalist in me anyway, but if others want to use it they can. Thanks for finding this!
